extensions/amp-list/0.1/amp-list.js

```diff
@@ -92,6 +94,12 @@ export class AmpList extends AMP.BaseElement {
 
     /** @private {?../../../src/ssr-template-helper.SsrTemplateHelper} */
     this.ssrTemplateHelper_ = null;
+
+    /** @private */
+    this.resizeExperimentOn_ = isExperimentOn(this.win, 'amp-list-resize');
```

Comment:
Nit: Move back to `render_()` as a local var there. For testing, it's nice to be able to toggle this experiment without refreshing the page.

Reply:
Oh gotcha. Sorry I misunderstood your original comment.

Reply:
Done.